### PR TITLE
minor: update path to kernel Documentation/.../sysctl.rst file

### PR DESCRIPTION
**Pull request by kaiwan**
The path shown to the kernel doc is outdated; let's update it.

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -10,11 +10,6 @@ jobs:
       - uses: actions/checkout@v2
       - name: docker build
         run: docker build -t slimm609/checksec:latest .
-      - name: Login to DockerHub
```
Comment (slimm609) [inline .github/workflows/push.yml:13]: why is this being removed?

Comment (kaiwan) [reply .github/workflows/push.yml:13]: Apologies, I don't mean to touch the push.yml or related files... with the GitHub workflow failing I was trying out something. Pl ignore it.
The first part 'minor: update path to kernel Documentation/.../sysctl.rst file' is the relevant patch only.

Comment (slimm609) [reply .github/workflows/push.yml:13]: Please rebase and remove that part. I can’t merge 1/2 of a merge request

**Comment by slimm609** [comment]
Closing in favor of #227

**Pull request closed without merging**